### PR TITLE
BaSP-TG-70: Setting wdio - Agustin Gatto

### PR DESCRIPTION
**Pull request by agusgatto8**

### CODE REVIEW

File: test/pageobjects/login.page.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable prettier/prettier */
```
Comment (JulianFloresDev) [inline test/pageobjects/login.page.js:1]: No recommend disable eslint

File: test/specs/super-admin-add-new.e2e.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable prettier/prettier */
```
Comment (JulianFloresDev) [inline test/specs/super-admin-add-new.e2e.js:1]: Same as above

File: test/specs/super-admin-add-new.e2e.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+       await expect(browser).toHaveUrl('https://alfon-a-trackgenix-app.vercel.app/home');
+    //    await SuperAdminPage.selectAdmins.click()
+    //    await expect(browser).toHaveUrl('https://alfon-a-trackgenix-btjyyqz4g-basp-a2022.vercel.app/admins');
+    //    await AdminsListPage.addButton.click();
+    //    await expect(browser).toHaveUrl('https://alfon-a-trackgenix-btjyyqz4g-basp-a2022.vercel.app/admins/form/0');
+    //    await FormPage.dataadmin('Test', 'Automation', 'algo@algo.com', 'Abcd1234', '47112225', '77547774', 'Lugar 1234');
+    //    await expect(FormPage.successfullyMessage).toHaveTextContaining('Admin created successfully!');
+    //    await FormPage.confirmButton.click();
+    //    await SuperAdminPage.logOutButton.click();
+    //    await browser.refresh();
```
Comment (JulianFloresDev) [inline test/specs/super-admin-add-new.e2e.js:25]: Comment lines here, be carefull with this!

File: test/specs/super-admin-delete.e2e.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+//         await AdminsListPage.deleteButton.click();
+//         await AdminsListPage.modalDelete.waitForDisplayed({timeout: 2000});
+//         await AdminsListPage.confirmDelete.click();
+//         await AdminsListPage.successMessage.waitForDisplayed({timeout: 1000});
+//         await expect(AdminsListPage.successMessage).toHaveTextContaining('Admin Deleted Successfully!');
+//         await AdminsListPage.exitModal.click();
+//         await SuperAdminPage.logOutButton.click();
+//         await browser.refresh();
+//     });
+// });
```
Comment (JulianFloresDev) [inline test/specs/super-admin-delete.e2e.js:27]: More comments lines

**Review by JulianFloresDev — COMMENTED** [review]
There are a lot of comments lines, i think you'll uncomment in future, rigth?

**Review by JulianFloresDev — DISMISSED** [review]

**Review by MatiasPazConti — APPROVED** [review]

**Review by JulianFloresDev — APPROVED** [review]

**Review by MatiasPazConti — APPROVED** [review]

**Pull request merged**